docs/reference/argocd.md

```diff
@@ -337,7 +337,7 @@ The following properties are available for configuring High Availability for the
 Name | Default | Description
 --- | --- | ---
 Enabled | `false` | Toggle High Availability support globally for Argo CD.
-RedisProxyImage | `haproxy` | The Redis HAProxy container image. This overrides the `ARGOCD_REDIS_HA_IMAGE`.
```

Comment:
We are not removing `ARGOCD_REDIS_HA_IMAGE`, right?

Reply:
No, it's still there.

Reply:
Can you please double check?  It looks like it has been replaced from my end.

Reply:
I didn't make any changes to where `ARGOCD_REDIS_HA_IMAGE` is used.  The doc above was wrong, `ARGOCD_REDIS_HA_IMAGE` was never used to specify the haproxy image, it was always used to specify the Redis image to use when in HA mode.  I had to introduce a new environment variable to allow the haproxy image to be specified.

The `ARGOCD_REDIS_HA_IMAGE` env var is used in the `getRedisHAContainerImage` function in  `pkg/controller/argocd/util.go`, which this PR does not modify

There is no way to override the Redis image or the Redis HA image in the ArgoCD CR